File: Unit5Sprint1/app/src/main/java/com/example/unit_5sprint1/retrofit/Network.java
package com.example.unit_5sprint1.retrofit;
import okhttp3.OkHttpClient;
import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class Network {

    private static final String BASE_URL = " https://api.tvmaze.com/";

    public static Retrofit getRetrofit(){
        return new Retrofit.Builder()
                .baseUrl(BASE_URL)
                .addConverterFactory(GsonConverterFactory.create())
                .client(new OkHttpClient.Builder().build())
                .build();
    }
}

File: AndroidMemesappusingMVVM/app/src/main/java/com/example/androidmemesappusingmvvm/remote/Network.kt
package com.example.androidmemesappusingmvvm.remote

import okhttp3.OkHttpClient
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory


object Network {
   fun getRetrofit():Retrofit{
       return Retrofit.Builder()
           .baseUrl("https://api.imgflip.com/")
           .addConverterFactory(GsonConverterFactory.create())
           .client(OkHttpClient())
           .build()
   }

}
